content/20.pharmaceuticals.md

```diff
@@ -7,12 +7,14 @@
 ### Introduction
 
 The novel coronavirus _Severe acute respiratory syndrome-related coronavirus 2_ (SARS-CoV-2) emerged in late 2019 and quickly precipitated the worldwide spread of novel coronavirus disease 2019 (COVID-19).
+<!--I never know whether to emphasize that this is the CFR and not the IFR. Both are also time and population-dependent...-->
```

Comment:
IFR stand for?

Reply:
Sorry I missed this until today! CFR = case fatality ratio, IFR = infection fatality ratio. "Case" means person diagnosed with COVID-19, "infection" means person infected with SARS-CoV-2. For most infectious diseases, CFR > IFR, because some people are asymptomatic. Also, as we do more testing, CFR generally decreases, as more people are diagnosed.

Reply:
```suggestion
```